### DISCUSSION TITLE
How to interpret Prophet vars weekday and season

### DISCUSSION THREAD

**Discussion by CarlosalbCb** [facebookexperimental/Robyn#521] [Q&A]
Hello, I'm getting negative response in the waterfall chart for the prophet vars weekday and season. 
How can I interpret these values for a retail store?

<img width="713" alt="Captura de Pantalla 2022-10-31 a la(s) 8 21 37" src="https://user-images.githubusercontent.com/13626228/199047663-59122584-956e-4461-ade0-d7371aec3ae8.png">

**Comment by gedkitchener-CT** [comment]
@CarlosalbCb 

I would assume that spread over a year, the aggregated (summed) effect of `season` is zero (by design), and spread over a week, the aggregated (summed) effect of `weekday` is zero (by design).

So if your total time range isn't an integer number of years then the sum of day-by-day values in `season` are probably not going to sum to zero, and likewise if the total time range isn't an integer number of weeks, then the sum of day-by-day values in `weekday` is probably not going to sum to zero.

For example, if `weekday` (Monday to Sunday) attribution is `[-10, -10, -10, -10, -10, +30, +20]`, then while an integer number of weeks will always sum to zero, I hope you can see that partial weeks will not in this contrived example.

That's my theory - but the authors may correct me.
